nats_test.go

```diff
@@ -1392,6 +1415,28 @@ func TestUserCredentialsChainedFile(t *testing.T) {
 	nc.Close()
 }
 
+func TestUserCredentialsChainedFileNotFound(t *testing.T) {
+	if server.VERSION[0] == '1' {
+		t.Skip()
+	}
+	ts := runTrustTLSServer()
+	defer ts.Shutdown()
+
+	chainedFile := createTmpFile(t, []byte(chained))
+	os.Remove(chainedFile)
+
+	nc, err := Connect(fmt.Sprintf("tls://127.0.0.1:%d", TEST_PORT),
```

Comment:
This test does not seem to be related to host shuffling. Also, this test "works" because there is a single server. If you had 2 servers, then the initial connect will get an INFO from first server with second connect URL which means that library will then try to connect to the second IP which then would produce the x509 error message.
cc @wallyqs

Reply:
Will remove and submit in second PR.